Samples/AppLifecycle/Activation/cs-console-unpackaged/CsConsoleActivation/Program.cs

```diff
@@ -8,23 +8,25 @@
 using Windows.Storage;
 using Microsoft.Windows.AppLifecycle;
 using static CsConsoleActivation.NativeHelpers;
+using global::Microsoft.Windows.ApplicationModel.DynamicDependency;
 
 namespace CsConsoleActivation
 {
     class Program
     {
         // Windows App SDK version.
-        private static uint majorMinorVersion = 0x00010000;
-        private static string versionTag = "";
+        private static var majorMinorVersion = global::Microsoft.WindowsAppSDK.Release.MajorMinor;
+        private static var versionTag = global::Microsoft.WindowsAppSDK.Release.VersionTag;
+        private static var minVersion = new global::Microsoft.Windows.ApplicationModel.DynamicDependency.PackageVersion(Microsoft.WindowsAppSDK.Runtime.Version.UInt64);
 
         private static string executablePath;
         private static string executablePathAndIconIndex;
 
         static void Main(string[] args)
         {
             // Initialize Windows App SDK for unpackaged apps.            
-            int result = MddBootstrap.Initialize(majorMinorVersion, versionTag);
-            if (result == 0)
+            int result = 0;
+            if (!Bootstrap.TryInitialize(majorMinorVersion, versionTag, minVersion, out result))
```

Comment:
NIT: Can remove versionTag + minVersion and just use the overloaded API to get the default. Not like the actual values are listed here anyway (just symbolic names)